package.json

```diff
@@ -1,43 +1,49 @@
 {
-  "name": "business-logic-mock-proxy",
-  "version": "0.0.2",
-  "description": "Business Logic Mock Proxy",
-  "engines": {
-    "node": "= 0.10.x"
+  "name"         : "business-logic-mock-proxy",
+  "version"      : "0.1.0",
+  "description"  : "Business Logic Mock Proxy",
+  "private"      : true,
+  "homepage"     : "http://www.kinvey.com",
+  "bugs"         : "https://github.com/Kinvey/business-logic-mock-proxy/issues",
+  "license"      : "Apache-2.0",
+  "author"       : "Kinvey, Inc.",
+  "contributors" : [
+    "Michael Salinger <mjsalinger@kinvey.com>",
+    "Mark van Seventer <mark@kinvey.com>"
+  ],
+  "repository" : { "type": "git", "url": "Kinvey/business-logic-mock-proxy" },
+  "scripts"    : {
+    "start"   : "node .",
+    "pretest" : "coffeelint -f .coffeelint config lib test",
+    "test"    : "mocha test/",
+    "test-collectionAccess" : "mocha test/lib/collection-access.test.coffee test/lib/collection-access-commands/*",
+    "test-configuration"    : "mocha test/lib/configuration-management.test.coffee",
+    "test-email"            : "mocha test/lib/email.test.coffee",
+    "test-logging"          : "mocha test/lib/logging.test.coffee",
+    "test-push"             : "mocha test/lib/push.test.coffee"
   },
   "dependencies": {
-    "business-logic-local-dashboard": "git+ssh://git@github.com:Kinvey/business-logic-local-dashboard.git#0.0.2",
-    "JSONStream": "^0.10.0",
-    "async": "0.9.0",
-    "body-parser": "1.6.5",
-    "bower": "^1.3.12",
+    "async": "1.4.x",
```

Comment:
Can you pin these modules to specific versions? We've had problems with devs introducing breaking changes into patch releases... :-1: